index.html

```diff
@@ -62,31 +68,43 @@
     return (toUTC(end) - toUTC(start)) / msDay;
 }
 
+function addDays(date, days) {
+    return date.setDate(date.getDate()+days);
+}
+
 function aesday(date) {
-    var epoch = new Date('2017-12-21');
     return days(epoch,date)%8 == 0;
-    
 }
 
+
+function nextAesday(date) {
+    return addDays(epoch,(Math.floor(days(epoch,date)/8)+1)*8);  

```

Comment:
Wouldn't this mutate `epoch`? I don't think you want that constant to change, right?